presto-main/src/main/java/com/facebook/presto/sql/analyzer/ExpressionAnalyzer.java

```diff
@@ -1116,6 +1119,13 @@ else if (frame.getType() == GROUPS) {
                 }
             }
 
+            if (node.isIgnoreNulls()) {
+                if (!functionResolution.isWindowValueFunction(function)) {
+                    String warningMessage = createWarningMessage(node, "IGNORE NULLS does not work for aggregate and ranking window functions, this will cause queries to fail in future versions");
```

Comment:
```suggestion
                    String warningMessage = createWarningMessage(node, "IGNORE NULLS is not used for aggregate and ranking window functions.  This will cause queries to fail in future versions.");
```